common/lib/xmodule/xmodule/capa_module.py

```diff
@@ -263,7 +266,7 @@ def has_support(self, view, functionality):
         Returns whether the given view has support for the given functionality.
         """
         if functionality == "multi_device":
-            return all(
+            return self.problem_types and all(
```

Comment:
It because I'm handling the error above by returning an empty set on invalid xml errors, and I didn't want to mistakenly return `True` by evaluating `all()` of an empty set.